apps/src/code-studio/components/AssetManager.jsx

```diff
@@ -258,11 +258,13 @@ export default class AssetManager extends React.Component {
         <AssetRow
           {...this.defaultAssetProps(asset)}
           api={boundApi}
-          onChoose={() =>
-            this.props.assetChosen(
-              STARTER_ASSET_PREFIX + asset.filename,
-              asset.timestamp
-            )
+          onChoose={
+            this.props.assetChosen &&
```

Comment:
Might be worth throwing in a comment since this is a workaround.

Reply:
Nevermind. I didn't read line 264. Didn't realize assetChosen was a function and we were checking if it exists.